core/src/io/github/eng12020team24/project1/gamestates/LoseState.java

```diff
@@ -0,0 +1,89 @@
+package io.github.eng12020team24.project1.gamestates;
+
+import com.badlogic.gdx.Gdx;
+
+import com.badlogic.gdx.Screen;
+
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.Input.Keys;
+
+public class LoseState implements Screen{
+    private AuberGame game;
+    private TextureAtlas uiAtlas;
+    private Button playAgainButton;
+    private TextureRegion gameOver;
+    private int gameOverX;
+    private Button exitButton;
+    private ActualGame newGame;
+    private MenuState menu;
+
+    public LoseState(AuberGame game){
+        this.game=game;
+        uiAtlas = new TextureAtlas(Gdx.files.internal("UISpritesheet/uispritesheet.atlas"));
+        playAgainButton = new Button(Gdx.graphics.getWidth()/2 - 440, 128, uiAtlas.findRegion("PLAY_AGAIN_BUTTON"));
+        exitButton = new Button(Gdx.graphics.getWidth()/2 - 200, 0, uiAtlas.findRegion("EXIT_BUTTON"));
+        gameOver = new TextureRegion(uiAtlas.findRegion("GAME_OVER_IMAGE"));
+        gameOverX = Gdx.graphics.getWidth()/2 - 490;
```

Comment:
why are these magic numbers (e.g. the 490) here?  Are they related to the size of the game over image?

Reply:
The numbers are to do with the width of the image. I was attempting to centralise everything as much as possible